integration/simulation/transaction_injector.go

```diff
@@ -141,10 +141,10 @@ func (ti *TransactionInjector) Start() {
 		})
 	}
 
-	wg.Go(func() error {
+	/*wg.Go(func() error {
```

Comment:
is this intended?

Reply:
Will be uncommented again, just unclogging the sim tests step by step